domains/Settings/ManageReligion/Web/Controllers/PersonalizeReligionController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Settings\ManageReligion\Web\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Settings\ManageReligion\Services\CreateReligion;
+use App\Settings\ManageReligion\Services\DestroyReligion;
+use App\Settings\ManageReligion\Services\UpdateReligion;
+use App\Settings\ManageReligion\Web\ViewHelpers\PersonalizeReligionViewHelper;
+use App\Vault\ManageVault\Web\ViewHelpers\VaultIndexViewHelper;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Inertia\Inertia;
+use Inertia\Response;
+
+class PersonalizeReligionController extends Controller
+{
+    public function index(): Response
+    {
+        return Inertia::render('Settings/Personalize/Religions/Index', [
+            'layoutData' => VaultIndexViewHelper::layoutData(),
+            'data' => PersonalizeReligionViewHelper::data(Auth::user()->account),
+        ]);
+    }
+
+    public function store(Request $request): JsonResponse
+    {
+        $data = [
+            'account_id' => Auth::user()->account_id,
+            'author_id' => Auth::user()->id,
```

Comment:
Instead of `Auth::user()->id`, you should write:
```suggestion
            'author_id' => Auth::id(),
```
(we should do a global replace in the whole code)

Reply:
Why?

Reply:
because we can.
And it's easier to read/maintain

Reply:
done in #282